src/main/resources/META-INF/plugin.xml

```diff
@@ -135,6 +141,12 @@
                 <override-text place="GitMacheteToolbar"/>
             </action>
 
+            <action id="GitMachete.TraverseFromCurrentAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.toolbar.TraverseFromCurrentAction"
+                    icon="com.virtuslab.gitmachete.frontend.icons.MacheteIcons.TRAVERSE">
+                <override-text place="GitMacheteToolbar"/>
```

Comment:
The toolbar's becoming pretty crowded:

<img width="105" alt="image" src="https://user-images.githubusercontent.com/3383210/205508145-b50c32dc-6f56-4332-a892-1f07e289a34c.png">

How about moving the relatively rarely used discover to the end, e.g. before the ❓ ?

Reply:
on the other hand the discover seems to be very useful thing for newbies 🤔 maybe a switch of places b/w the traverse and the discover?

Reply:
Hmm okay let's start with switching places 🤔